test/WebSites/HtmlGenerationWebSite/Views/HtmlGeneration_Home/DuplicateAntiforgeryTokenRegistration.cshtml

```diff
@@ -0,0 +1,19 @@
+﻿@{
```

Comment:
Rename everything about this new test to reflect the actual scenario. Not testing any duplication, just multiple `<form>`s in a single view.